scripts/BaseL0FraxDVNScript.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: ISC
+pragma solidity ^0.8.19;
+
+import { Script } from "forge-std/Script.sol";
+import { DVN } from "@layerzerolabs/lz-evm-messagelib-v2/contracts/uln/dvn/DVN.sol";
```

Comment:
Thoughts on creating a simple `FraxDVN` contract to inherit `DVN` so that we have our Frax brand across the DVN?

Reply:
Addressed!